content/tutorial-deep-learning-on-mnist.md

```diff
@@ -72,20 +72,6 @@ data_sources = {
 **2.** Load the data. First check if the data is stored locally; if not, then
 download it.
 
-```{code-cell} ipython3
-:tags: [remove-cell]
-
-# Use responsibly! When running notebooks locally, be sure to keep local
-# copies of the datasets to prevent unnecessary server requests
-headers = {
-    "User-Agent": "Mozilla/5.0 (X11; Linux x86_64; rv:10.0) Gecko/20100101 Firefox/10.0"
-}
-request_opts = {
-    "headers": headers,
-    "params": {"raw": "true"},
-}
-```
-
```

Comment:
This isn't so much a workaround as necessary setup for downloading the data from an external source - it needs to be in place in order for the downloading to work (either on binder or for users running locally).

Reply:
Ah sorry, I misunderstood this!